Project/MyGameLibrary/Player.cs

```diff
@@ -1,13 +1,22 @@
-﻿using System;
+﻿using MyGameLibrary;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace Fall2020_CSC403_Project.code {
   public class Player : BattleCharacter {
+
+    private Inventory inventory;
+
     public Player(Vector2 initPos, Collider collider) : base(initPos, collider) {
+            inventory = new Inventory();
+    }
 
+    public Inventory GetInventory()
```

Comment:
Similar to the comment I made in the InventoryObject class, should just use a property here instead of a separate accessor function.

Reply:
This would cause a conflict where the class name is the same as the property.